src/docs.js

```diff
@@ -1,4 +1,35 @@
-function documenter() {
+let tar = require('tar-stream');
+let _ = require('lodash');
+let assert = require('assert');
+let fs = require('fs');
+let path = require('path');
+
+async function documenter(options) {
+
+  options = _.defaults({}, options, {
+    schemas: [],
+  });
+  assert(options.schemas, 'options.schemas must be given');
+  assert(options.schemas instanceof Array, 'options.schemas must be an array');
+
+  let schemas = options.schemas;
+  let tarball = tar.pack();
+
+  //add schemas to tarball
+  schemas.forEach(schema => {
+    let data = JSON.stringify(schema, null, 2);
+    tarball.entry({name: 'schema/' + schema.id}, data);
```

Comment:
we can make an option like `option.schemaIdPrefix` that is then removed the schema if it contains it...

That way we can support the current validator code, until we decide how to figure out how to do this right...